src/components/Fomulario/__test__/validate.test.ts

```diff
@@ -0,0 +1,104 @@
+import { ErrorObject } from "../../../validation/model";
+import { IMCFormModel } from "../Formulario";
+import * as validators from "../../../validation/validate";
+import { validateAltura, validateIMC, validatePeso } from "../validate";
+import * as validatorIMC from "../validate";
+
+const ALTURA = "altura";
+const PESO = "peso";
+
+describe("validate", () => {
+  describe("validateAltura", () => {
+    it("quando validateAltura for chamado, deve chamar o validateRequired com os parametros esperados", () => {
+      const values: IMCFormModel = { altura: "173" };
+      const spy = jest.spyOn(validators, "validateRequired");
+      var errors: ErrorObject<IMCFormModel> = {};
+      validateAltura(errors, values);
+      expect(spy).toHaveBeenCalled();
+      expect(spy).toHaveBeenCalledWith(
+        expect.anything(),
+        expect.anything(),
+        ALTURA
+      );
+    });
```

Comment:
```suggestion
      expect(spy).toHaveBeenCalledWith(
        expect.anything(),
        expect.anything(),
        ALTURA
      );
    });
```
O `toHaveBeenCalledWith` já contempla o `toHaveBeenCalled`